client/src/components/NotFound.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+export default function NotFound() {
+    return (
+        <div>
+            <h1>Page not found</h1>
+        </div>
+    )
+}
+
+// if user types in something that the url doesn't recognize, it will take them to this page. 
```

Comment:
This is a good thing to have as a catch for keeping people on pages that you want them on, but you would have to create the path on your front end router that would send the users to this page if they were to put in an invalid URL